yoti-sdk-parent/pom.xml

```diff
@@ -0,0 +1,419 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.yoti</groupId>
+  <artifactId>yoti-sdk-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>2.0.0-SNAPSHOT</version>
+  <name>Yoti SDK Parent Pom</name>
+  <description>Parent pom for the Java SDK projects</description>
+  <url>https://github.com/getyoti/yoti-java-sdk</url>
+  
+  <developers>
+    <developer>
+      <name>Andras Bulla</name>
+    </developer>
+    <developer>
+      <name>Radoslaw Busz</name>
+    </developer>
+    <developer>
+      <name>David Goaté</name>
+    </developer>
+    <developer>
+      <name>Attila Kiss</name>
+    </developer>
+    <developer>
+      <name>Quirino Zagarese</name>
+    </developer>
+    <developer>
+      <name>Michael Buck</name>
+    </developer>
+    <developer>
+      <name>Francesco Sorice</name>
+    </developer>
+    <developer>
+      <name>Vincenzo Candela</name>
+    </developer>
+  </developers>
+  
+  <licenses>
+    <license>
+      <name>Yoti License</name>
+      <url>https://github.com/getyoti/yoti-java-sdk/blob/master/LICENSE.md</url>
+    </license>
+  </licenses>
+  
+  <scm>
+    <connection>scm:http://github.com/getyoti/yoti-java-sdk.git</connection>
+    <developerConnection>scm:http://github.com/getyoti/yoti-java-sdk.git</developerConnection>
+    <url>http://github.com/getyoti/yoti-java-sdk.git</url>
+  </scm>
+  
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+  
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <yoti.java.sdk.version>2.0.0-SNAPSHOT</yoti.java.sdk.version>
+    <supported.java.version>1.6</supported.java.version>
+    <maven.compiler.source>${supported.java.version}</maven.compiler.source>
+    <maven.compiler.target>${supported.java.version}</maven.compiler.target>
+    
+    <slf4j.version>1.7.25</slf4j.version>
+    <bouncy.castle.version>1.59</bouncy.castle.version>
+    <jackson.version>2.7.9.1</jackson.version>
+    <google.protobuf.version>3.5.0</google.protobuf.version>
+    
+    <springboot.version>1.5.10.RELEASE</springboot.version>
+    <spring.core.version>4.3.14.RELEASE</spring.core.version>
+    <spring.security.version>4.2.3.RELEASE</spring.security.version>
+    <javax.servlet.version>2.5</javax.servlet.version>
+    
+    <junit.version>4.12</junit.version>
+    <mockito.version>2.8.47</mockito.version>
+    <hamcrest.version>1.3</hamcrest.version>
+    
```

Comment:
do you mind to add some session separator like `<-- Maven plugins version -->`? thx

Reply:
I've made some changes - could you look again please, see if that's better?